couchpotato/core/media/movie/providers/info/couchpotatoapi.py

```diff
@@ -102,12 +106,141 @@ def getInfo(self, identifier = None, adding = False, **kwargs):
 
         return {}
 
+
     def getReleaseDate(self, identifier = None):
         if identifier is None: return {}
 
         dates = self.getJsonData(self.urls['eta'] % identifier, headers = self.getRequestHeaders())
-        log.debug('Found ETA for %s: %s', (identifier, dates))
 
+        #This grabs release date info from omdbapi/rottentomatoes
+        temp2 = self.getJsonData("http://www.omdbapi.com/?i=%s&tomatoes=true&plot=short&r=json" % identifier)
+        title = temp2['Title']
+        year = int(temp2['Year'])
+
+        ddate=0 #throw away what couchpotatoai is returning since it is garbage at this time
+        tdate=0
+        dvd_date= temp2['DVD']
+        theater_date=temp2['Released']
+        if theater_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            tdate=int(time.mktime(time.strptime(theater_date,p)))
+
+        if dvd_date != 'N/A' and year >1972:
+            p='%d %b %Y'
+            ddate=int(time.mktime(time.strptime(dvd_date,p)))
+           
+        if (ddate !=0):    
+            if ddate < tdate:
+                ddate = 0 #if the dvd release date occurs before the theater release date, assume the data is wrong
+                tdate = 0
+
+        if (ddate or tdate):
```

Comment:
I see what you mean ---> changing to 'and' will mean it will accept OMDBAPI release date only if both theater and dvd dates are found... 

I think the reason I put "OR" was because in practice I found that if one was missing on one service, the probability of it being incorrect on the other service was high... and when I was writing the code I preferred no date over and incorrect date. 

Also, sometimes OMDBAPI would have partial information lacking in THEMOVIEDB.. 

I mean code can be written to cache the data and choose the most appropriate result in the end. I just didnt bother.. I will change it to "and" and try it out. and maybe write some improvements in the next few weeks.. so that we dont throw away perfectly good data from OMDBAPI because THEMOVIEDB is lacking.. or vice versa.. etc.

Thinking about it now and having been using this feature for a while, I suppose what you are suggesting makes more sense.. If a date is incorrect then someone could report it to the offending service (I did this a couple times and they did update their db).. I'll commit the change later this evening..

Please let me know if you have any further suggestions.. The Netflix feature was working flawlessly for the past several months until a few days ago when it seemed allflicks implemented some kind of cookie support. I checked in the update to "follow" allflicks change .. my ability to check netflix is totally dependent on allflicks....